src/SDKs/AzSdk.test.reference.props

```diff
@@ -3,7 +3,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Test.HttpRecorder" Version="[1.7.0,2.0.0)" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="[1.6.0,2.0.0)" />
-    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="[3.3.5,4.0.0)" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="[3.3.6,4.0.0)" /> 
```

Comment:
@brjohnstmsft any reason you needed the new version of azure but not new version of clientruntime?
I would like to understand what prompted you to take dependency on the new version?